Standards/scs-0217-v1-cluster-hardening.md

```diff
@@ -0,0 +1,470 @@
+By default, a request would be denied.
+
+Kubernetes provides several authentication modes to authorize a request:
+
+- *Node*
+
+  The [Node authorization mode][node-authorization] grants permission to a Kubelet
+  based on the scheduled pods running on them. It allows a Kubelet to perform specific
+  API operations. The goal is to have a minimal set of permissions to ensure
+  the Kubelet can operate correctly.
+  Each Kubelet identifies with credentials belonging to the `system:nodes` group and
+  a username `system:nodes:<node>` against this authorizer.
+
+- *ABAC (Attribute-based access control)*
+
+  ABAC grants access rights based on policies dependent on attributes like
+  user attributes, resource attributes or environment attributes.
+  An example would be the `resource` attribute, which could limit access for a user
+  to only `Pod` resources.
+
+- *RBAC (Role-based access control)*
+
+  RBAC is a method of regulating access to the resources based on the roles of
+  individual users. A user therefore must have the ability to perform a specific set
+  of tasks with a set of resources based on his role.
+  Kubernetes implements `Role`s to accomplish this and binds these with `Role Binding`s
+  to a user in order to specify his permission set.
+
+- *Webhook*
+
+  Webhook authorization uses an HTTP callback to check the authorization of a user
+  against a URL provided for this mode. This externalises the authorization part
+  outside of Kubernetes.
+
+Most organizations and deployments work with RBAC, most often due to organizational or
+customer-owner-relationship-like structures in place.
+Nonetheless, neither ABAC, RBAC nor Webhook authorization can be recommended over the
+other, since this all depends on the use case and required structure of a deployment.
+Using at least one of these modes is recommended.
+
+It is also recommended to enable the Node authorizer in order to limit Kubelet
+permissions to a minimum operational state.
+
+#### Admission Controllers
+
+Admission controllers intercept requests to the Kubernetes API after the
+authentication and authorization step, which validate and/or mutate the request.
+This step is limited to `create`, `modify` and `delete` objects as well as custom
+verbs, but other requests are not blocked.
+Kubernetes provides multiple admission controllers, some of which are enabled by default.
+
+One recommended admission controller is the [`NodeRestriction` controller][node-restriction],
+which limits the `Node` and `Pod` objects a Kubelet is allowed to modify to their own `Node` or
+objects that are bound to them. It also disallows updating or removing taints and prevents changing
+or adding labels with a `node-restriction.kubernetes.io/` prefix.
+Be aware that Kubelets will only be limited by this admission controller, if the use credentials
+in the `system:nodes` group with a `system:node:<nodeName>` username. Administrators must therefore
```

Comment:
```suggestion
Be aware that Kubelets will only be limited by this admission controller, if the user credentials
in the `system:nodes` group begin with a `system:node:<nodeName>` username. Administrators must therefore
```
not sure this is the intended meaning?

Reply:
Yes, thanks for the catch.
[6805ea0](https://github.com/SovereignCloudStack/standards/pull/581/commits/6805ea0289d5595323c73ed5df8b29d9e90fd889)